DirectGuiDesigner/core/CustomWidgets.py

```diff
@@ -3,11 +3,13 @@
 import json
 import importlib
 import pathlib
+import sys
 import types
 from panda3d.core import ConfigVariableString
 from DirectGuiDesigner.core.WidgetDefinition import PropertyEditTypes, Definition, DEFINITIONS
 
-class CustomWidget():
+
+class CustomWidget:

```

Comment:
Is there a reason why you removed the brackets? Is that a pep recommendation?

Reply:
I mainly think it is a bit ugly to have redundant parentheses, but I don't know if that is a pep recommendation. However I do not really care that much about it

Reply:
Ok, I haven't found much details about this yet but it seems pythons official code also does that so, I'm fine with this change. But, if it's done here, we should also change other parts where applicable to have the code base consistent.

Reply:
Yes, there are some other places where this occurs. I will add a commit to change that.